core/config/envvar/schema.go

```diff
@@ -274,6 +274,9 @@ type ConfigSchema struct {
 	ClientNodeURL        string `env:"CLIENT_NODE_URL" default:"http://localhost:6688"`
 	InsecureSkipVerify   bool   `env:"INSECURE_SKIP_VERIFY" default:"false"`
 
+	// VRF
+	VRFBatchFulfillmentsFeatureEnabled bool `env:"VRF_BATCH_FULFILLMENTS_FEATURE_ENABLED" default:"false"`
```

Comment:
Not sure which is better, but did you consider putting this in the jobspec instead of node level config?

Reply:
Heh, I don't mind that at all, since it would mean not touching the config mess that we have.